packages/common/src/ui/components/errors/GenericErrorFallback.tsx

```diff
@@ -0,0 +1,25 @@
+import { Box, Typography, Button } from '@material-ui/core';
+import React, { FC } from 'react';
+import { ErrorBoundaryFallbackProps } from './types';
+import UnhappyMan from '../../icons/UnhappyMan';
+
+export const GenericErrorFallback: FC<ErrorBoundaryFallbackProps> = ({
+  onClearError,
+}) => {
+  return (
+    <Box
+      display="flex"
+      flexDirection="column"
+      justifyContent="center"
+      alignItems="center"
+      height="100%"
+      flex={1}
+    >
+      <UnhappyMan />
+      <Typography variant="h3">Oops! Somethings gone wrong.</Typography>
```

Comment:
this looks better if the `Typography` component has 20px of padding on it
and a localisation for it.. please?

Reply:
I also think it should be `Something's gone wrong` with the apostrophe as a contraction for 'has'